app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -519,6 +532,23 @@ private void configureRightCard() {
         rightCardMapButton.setOnClickListener(v -> presenter.openCoordinateMap());
     }
 
+    public void configurePrevButton(){
+        prevTitleDecs.setOnClickListener((View v) -> {
+            BasicKvStore storePrevTitleDesc = new BasicKvStore(this, "prevTitleDesc");
+            String title = storePrevTitleDesc.getString("title");
+            Title t = new Title();
+            t.setTitleText(title);
+
+            Description description= new Description();
+            List<Description> finalDesc = new LinkedList<>();
+            String desc = storePrevTitleDesc.getString("description0");
```

Comment:
Set descriptions in all languages that are saved. ie for all `description_<language_code>`.